src/vanilla/mod.rs

```diff
@@ -0,0 +1,35 @@
+use std::env;
+use std::path::PathBuf;
+
+fn var(key: &str) -> String {
+    match env::var(key) {
+        Ok(val) => val,
+        Err(err) => panic!("couldn't find env var {}; err: {:?}", key, err)
```

Comment:
No, an envar being unset is very much possible. This function should be removed and calls to it replaced with calls to `env::var` and proper error handling.